views/base.html

```diff
@@ -55,6 +55,9 @@
     </div> -->
 
     <main>
+      {% block communities %}
```

Comment:
Will be addressed when https://github.com/18F/doi-extractives-data/commit/8fbdbc246b4e68e11dec3c225e8790b125990327 merges.